hugo.toml

```diff
@@ -96,13 +96,15 @@ github_branch = "main"
 # Enable Lunr.js offline search
 offlineSearch = true
 
+url_latest_version = "https://kairos.io/v3.3.1/docs"
+development = true
+
 [params.softwareVersions]
 # renovate: datasource=github-releases depName=kairos-io/provider-kairos
 provider_version = "v2.9.2"
 # renovate: datasource=github-releases depName=k3s-io/k3s
 k3s_version = "v1.32.1+k3s1"
-# renovate: datasource=github-releases depName=kairos-io/kairos
-kairos_version = "v3.3.1"
+kairos_version = "master"
```

Comment:
otherwise the "dev" documentation, points to a stable release which might be confusing